.php-cs-fixer.dist.php

```diff
@@ -12,15 +12,15 @@
  * with this source code in the file LICENSE.
  */
```

Comment:
> > if `INPUT` and `EXPECTED` should be added to the list of reserved markers (really common in test cases)
> 
> also discussed - https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7660#discussion_r1441851523

Reply:
I understand concern if somoene would be maintaining it, but I also see value in having them

I'm ok for both